dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/rest/DssHttpClientServiceActor.java

```diff
@@ -0,0 +1,39 @@
+package io.github.ztkmkoo.dss.core.actor.blocking.rest;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import io.github.ztkmkoo.dss.core.message.blocking.DssBlockingRestCommand;
+
+/**
+ * @author Kebron ztkmkoo@gmail.com
+ * @create 2020-08-13 01:56
+ */
+public class DssHttpClientServiceActor<R extends DssBlockingRestCommand.HttpRequest> {
+
+    public static <R extends DssBlockingRestCommand.HttpRequest> Behavior<DssBlockingRestCommand> create(DssHttpClientService<R> service, Class<R> requestClass) {
```

Comment:
I'm a little confused, is it okay to sort like "<R, variable, and constructor", not sort like "variable, constructor, and <R"?
Maybe I have to redo #36 ...

Reply:
Hi, it looks i miss some details bout class format... 
I'm so sorry.

i mentioned it at #36 issue...
``` java
class Foo {

    // static fields
    // static initializer
    // static methods
    // instance fields
    // constructors
    // instance methods
}
```